src/frontend/src/components/icons.ts

```diff
@@ -357,3 +389,12 @@ export const lockIcon = html`<svg
     />
   </g>
 </svg>`;
+
+export const lockedIcon = html`
+  <svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M3.99996 14.6666C3.63329 14.6666 3.31929 14.536 3.05796 14.2746C2.79663 14.0133 2.66618 13.6995 2.66663 13.3333V6.66663C2.66663 6.29996 2.79729 5.98596 3.05863 5.72463C3.31996 5.46329 3.63374 5.33285 3.99996 5.33329H4.66663V3.99996C4.66663 3.07774 4.99174 2.29152 5.64196 1.64129C6.29218 0.991071 7.07818 0.666182 7.99996 0.666626C8.92218 0.666626 9.7084 0.991738 10.3586 1.64196C11.0088 2.29218 11.3337 3.07818 11.3333 3.99996V5.33329H12C12.3666 5.33329 12.6806 5.46396 12.942 5.72529C13.2033 5.98663 13.3337 6.3004 13.3333 6.66663V13.3333C13.3333 13.7 13.2026 14.014 12.9413 14.2753C12.68 14.5366 12.3662 14.6671 12 14.6666H3.99996ZM3.99996 13.3333H12V6.66663H3.99996V13.3333ZM7.99996 11.3333C8.36663 11.3333 8.68063 11.2026 8.94196 10.9413C9.20329 10.68 9.33374 10.3662 9.33329 9.99996C9.33329 9.63329 9.20263 9.31929 8.94129 9.05796C8.67996 8.79663 8.36618 8.66618 7.99996 8.66663C7.63329 8.66663 7.31929 8.79729 7.05796 9.05863C6.79663 9.31996 6.66618 9.63374 6.66663 9.99996C6.66663 10.3666 6.79729 10.6806 7.05863 10.942C7.31996 11.2033 7.63374 11.3337 7.99996 11.3333ZM5.99996 5.33329H9.99996V3.99996C9.99996 3.4444 9.80552 2.97218 9.41663 2.58329C9.02774 2.1944 8.55552 1.99996 7.99996 1.99996C7.4444 1.99996 6.97218 2.1944 6.58329 2.58329C6.1944 2.97218 5.99996 3.4444 5.99996 3.99996V5.33329Z"
+      fill="#11B10E"
```

Comment:
Should be either be `currentColor` or not set

Reply:
@nmattia `currentColor` preferably. none would break the `[fill]`css selector